panther_analysis_tool/main.py

```diff
@@ -179,11 +179,14 @@ def load_module(filename: str) -> Tuple[Any, Any]:
     return module, None
 
 
-def load_analysis_specs(directories: List[str]) -> Iterator[Tuple[str, str, Any, Any]]:
+def load_analysis_specs(
+    directories: List[str], ignored_files: List[list]
```

Comment:
Isn't `ignored_files` a list of strings?

```suggestion
    directories: List[str], ignored_files: List[str]
```